### ISSUE TITLE
Bubble wrap bug

### ISSUE THREAD

**Issue by IRXOSM** [TymanWasTaken/tyman.tech#33]
the [bubble wrap](https://tyman.tech/bubbles) doesn't play all sound when clicking a lot of bubbles quickly.

**Comment by TymanWasTaken** [comment]
Yeah I made it like 2 years ago and I did have a better version but couldn't find it

**Comment by IRXOSM** [comment]
Oh ok. [Here is an alternative](https://www.youtube.com/watch?v=dQw4w9WgXcQ).

**Comment by TymanWasTaken** [comment]
![Uploading Screenshot_20210423-152305.png…]()
How about no

**Comment by IRXOSM** [comment]
I was just taking my revenge 
![image](https://user-images.githubusercontent.com/79060441/115931387-4df14900-a48b-11eb-80cc-ff150aeb67cd.png). (bubble wrap bug still real tho)
